django_netjsonconfig/urls.py

```diff
@@ -1,9 +1,8 @@
 from django.conf.urls import url
-
-from . import views
+from django.contrib import admin
 
 app_name = 'django_netjsonconfig'
 
 urlpatterns = [
-    url(r'^netjsonconfig/schema\.json$', views.schema, name='schema'),
+    url(r'admin/', admin.site.urls),
```

Comment:
we should not put the admin URLs here because most existing OpenWISP based projects already declare the admin URLs in the root urlconf, even the test django project of this repo:
https://github.com/openwisp/django-netjsonconfig/blob/master/tests/urls.py#L7

I think you need to leave an empty `urlpatterns` here (`urlpatterns = []`) and add a comment mentioning it's done for backward compatibility

Reply:
Got it! Made the required change! :smile: